face/FaceSelection.js

```diff
@@ -55,5 +56,6 @@ function start(faces, viewmode) {
   // return the best face available
   // eslint-disable-next-line
   let best = bf.reduce((max, p) => p.rate > max.rate ? p : max, bf[0]);
+  console.log(best);
```

Comment:
Could you remove this log for the release?

Reply:
af119e2